vendors/espressif/boards/esp32/aws_tests/config_files/aws_iot_network_config.h

```diff
@@ -52,6 +52,6 @@
  *
  */
 
-#define configENABLED_NETWORKS      ( AWSIOT_NETWORK_TYPE_WIFI | AWSIOT_NETWORK_TYPE_BLE )
+#define configENABLED_NETWORKS      ( AWSIOT_NETWORK_TYPE_WIFI )
```

Comment:
Why is this change required?

Reply:
For the tests, espressif does not have enough stack to enable both HTTP and BLE together and throws compile time error.